use fmt::format until C++20 support for std::format is available

Index: rts/System/FileSystem/Archives/PoolArchive.cpp
--- rts/System/FileSystem/Archives/PoolArchive.cpp.orig
+++ rts/System/FileSystem/Archives/PoolArchive.cpp
@@ -11,6 +11,8 @@
 #include <iostream>
 #include <format>
 
+#include <fmt/core.h>
+
 #include "System/FileSystem/DataDirsAccess.h"
 #include "System/FileSystem/FileSystem.h"
 #include "System/Threading/SpringThreading.h"
@@ -192,12 +194,12 @@ std::string CPoolArchive::GetPoolFileName(const std::a
 
 	const std::string prefix(c_hex    ,  2);
 	const std::string pstfix(c_hex + 2, 30);
-	return std::format("{}/{}.gz", prefix, pstfix);
+	return fmt::format("{}/{}.gz", prefix, pstfix);
 }
 
 std::string CPoolArchive::GetPoolFilePath(const std::string& poolRootDir, const std::string& poolFile)
 {
-	std::string rpath = std::format("{}/pool/{}", poolRootDir, poolFile);
+	std::string rpath = fmt::format("{}/pool/{}", poolRootDir, poolFile);
 	return FileSystem::FixSlashes(rpath);
 }
 
